File: Proyecto_P1.Lib/Exceptions/VehiculoException.cs
namespace Proyecto_P1.Lib.Exceptions
{
    /// <summary>
    /// Excepción personalizada para errores relacionados con operaciones de vehículos.
    /// </summary>
    public class VehiculoException : Exception
    {
        public VehiculoException(string message) : base(message) { }
        public VehiculoException(string message, Exception innerException) : base(message, innerException) { }
    }
}

File: ProyectoEstructuras1/src/main/java/com/mycompany/proyectoestructuras1/VehiculoException.java
package com.mycompany.proyectoestructuras1;

/**
 * Excepción personalizada para errores relacionados con operaciones de vehículos.
 */
public class VehiculoException extends Exception {

    public VehiculoException(String message) {
        super(message);
    }

    public VehiculoException(String message, Throwable cause) {
        super(message, cause);
    }
}
